Resources/Prototypes/_Floof/Entities/Clothing/Back/duffels.yml

```
@@ -0,0 +1,27 @@
+- type: entity
+  parent: ClothingBackpack
+  id: ClothingBackpackDuffelCourierDevy
+  name: Express Courier Trunk
+  description: A Massive storage trunk for express space couriers. It has a large inside at the cost of being heavy.
```

This could be rewritten to sound more correct, especially at the "large inside" part. Here is a suggestion. Note that you can rewrite this suggestion however you like, and it's intended to give you an idea.
```suggestion
  description: A massive storage trunk issued to express space couriers. It boasts a spacious interior but is considerably heavy.
```